SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -49,11 +36,17 @@ FLocalDeploymentManager::FLocalDeploymentManager()
 	, bStartingSpatialService(false)
 	, bStoppingSpatialService(false)
 {
+}
+
+void FLocalDeploymentManager::PreInit(bool bChinaEnabled)
+{
+#if PLATFORM_WINDOWS
```

Comment:
why is this only enabled for windows? we want local deployments for mac as well :)

Reply:
Oof that came from merging, I chose the wrong one. So both Init and PreInit can be run on mac now yah?

Reply:
local deployment manager should work on mac. if it doesn't it's a bug